packages/ui-test-queries/src/utils/bindElementToEvents.ts

```diff
@@ -278,7 +283,7 @@ async function fireKeyboardEvent(
             fireEvent({
               ...init,
               key,
-              which: keyCode,
+              which: keyCode, // TODO remove it, its deprecated
               keyCode
             })
```

Comment:
While trying to fix the focus and blur tests failing I made the typing of these a bit better